### PR TITLE
Expose a way for unpackaged apps to persist window location

### PR DESCRIPTION
**Pull request by dotMorten**
#61
@mikebattista would this work for you?

### CODE REVIEW

File: src/WinUIEx/WindowManager.cs [hunk 1]
```diff
@@ -231,27 +240,38 @@ private struct MINMAXINFO
         /// <remarks>
         /// The ID must be set before the window activates. The window size and position
         /// will only be restored if the monitor layout hasn't changed between application settings.
-        /// The property uses ApplicationData storage, and therefore is currently only functional for
-        /// packaged applications.
+        /// By default the property uses ApplicationData storage, and therefore is currently only functional for
+        /// packaged applications. If you're using an unpackaged application, you must also set the <see cref="PersitanceStorage"/>
+        /// property and manage persisting this across application settings.
         /// </remarks>
+        /// <seealso cref="PersitanceStorage"/>
```
Comment (mikebattista) [inline src/WinUIEx/WindowManager.cs:247]: Typo here.

**Comment by mikebattista** [comment]
I'll need to try it and see what the experience is for the app.

**Comment by dotMorten** [comment]
I'm open for other ways to expose it - this was just the simplest way I could think of.

**Comment by mikebattista** [comment]
I understand the loading piece. If the app had previously saved the dictionary to LocalAppData, on startup it can read it and set it for PersistenceStorage, then you will load the window size and position as you do today without requiring the app to do its own P/Invokes. That's good.

For save, I'm assuming you'll be writing to the provided Dictionary in memory whenever the window size and position changes? If so, what is the expectation on the developer to save this? Do they need to subscribe to an event to know when save? You probably don't want to write to disk for every pixel change right?

**Comment by dotMorten** [comment]
> For save, I'm assuming you'll be writing to the provided Dictionary in memory whenever the window size and position changes?

This happens when the window closes. You could create a custom `IDictonary<string,object>` class that'll just read from/write to disk (or just pass it your settings dictionary as-is)
(read is happening on the first window activated event).

**Comment by dotMorten** [comment]
@mikebattista Any more thoughts on this approach?

**Comment by mikebattista** [comment]
I'll need to try it out but am focused on shipping the 5.2 Template Studio update at the moment. Will take a look for the next update.

**Comment by dotMorten** [comment]
Closing this in favor of https://github.com/dotMorten/WinUIEx/commit/5e179c280788be8f32fa3c95401d741ebcd3c822

**Pull request closed without merging**